README.md

```diff
@@ -230,6 +230,7 @@ ReactDOM.render(
     document.getElementById('root')
 );
 ```
+Functional Components can cause unnecessary render calls. In such cases consider using the [PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent) or the [shouldComponentUpdate method](https://reactjs.org/docs/react-component.html#shouldcomponentupdate).
```

Comment:
missed comma: "In such cases, consider..."

Reply:
Thanks, fixed